server/controllers/users.js

```diff
@@ -264,6 +265,48 @@ class UsersController {
       }).catch(next);
   }
 
+  /**
+   * @description Static method for admin to get a list of all users
+   * @param  {object} req body of the Admin's request
+   * @param  {object} res  body of the response message
+   * @param  {function} next next function to be called
+   * @returns {object} The body of the resposne message
+   */
+  static adminGetUsers(req, res, next) {
```

Comment:
there is already a get all profiles function. edit it to confirm admin token of the requester and add the results you want.

Reply:
Thanks, I have fixed this